.github/workflows/debian.yml

```diff
@@ -0,0 +1,72 @@
+# This action originally modeled off of the Debian build action:
+# https://salsa.debian.org/wouter/ola/-/blob/a38a396f6994b2b1af8efec9e208aee4e67f77aa/.gitlab-ci.yml
+
+name: debian
+on: [push, pull_request]
+jobs:
+  debian-build:
+    name: 'Debian Build ${{ matrix.image_tag }} amd64'
```

Comment:
Interesting that one can use the `matrix` variable before having defined the actual matrix. Nice!